interface/forms/telemed/view.php

```diff
@@ -0,0 +1,378 @@
+<?php
+/**
+ * transfer summary form.
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Naina Mohamed <naina@capminds.com>
+ * @copyright Copyright (c) 2012-2013 Naina Mohamed <naina@capminds.com> CapMinds Technologies
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+
+
+require_once("../../globals.php");
+require_once("$srcdir/api.inc");
+require_once("$srcdir/patient.inc");
+require_once("$srcdir/options.inc.php");
+
+require_once("$srcdir/MedEx/API.php");
+
+use OpenEMR\Common\Csrf\CsrfUtils;
+use OpenEMR\Core\Header;
+use OpenEMR\OeUI\OemrUI;
+
+$MedEx = new MedExApi\MedEx('MedExBank.com');
+
+$form_name   = "telemed";
+$form_folder = "telemed";
+$Form_Name   = "TeleHealth Visit";
+$action      = $_REQUEST['action'];
+$id          = $_REQUEST['id'];
+$display     = $_REQUEST['display'];
+$pid         = $_SESSION['pid'];
+$refresh     = $_REQUEST['refresh'];
+
+
+formHeader("TeleHealth Visit");
+$returnurl = 'encounter_top.php';
+$formid = 0 + (isset($_GET['id']) ? $_GET['id'] : 0);
+$obj = $formid ? formFetch("form_telemed", $formid) : array();
+
+$query          = "SELECT * FROM patient_data where pid=?";
+$pat_data       =  sqlQuery($query, array($pid));
+
+$query10 = "select  *,form_encounter.date as encounter_date
+               from forms,form_encounter,form_telemed
+               where
+                    forms.deleted != '1'  and
+                    forms.formdir='telemed' and
+                    forms.encounter=form_encounter.encounter  and
+                    forms.form_id=form_telemed.id and
+                    form_telemed.id =? ";
+
+$encounter_data =sqlQuery($query10, array($id));
+$tm_duration = $encounter_data['tm_duration'];
+$tm_subj = $encounter_data['tm_subj'];
+$tm_obj = $encounter_data['tm_obj'];
+$tm_imp = $encounter_data['tm_imp'];
+$tm_plan = $encounter_data['tm_plan'];
+
+$dated = new DateTime($encounter_data['encounter_date']);
+$dated = $dated->format('Y-m-d');
+$visit_date = oeFormatShortDate($dated);
+
+$query = "select * from openemr_postcalendar_events where pc_pid=? and pc_eventDate=?";
+$appt = sqlQuery($query, array($pid,$encounter_data['encounter_date']));
+$reason = $appt['pc_hometext'];
+
+$query          = "SELECT * FROM users where id = ?";
+$prov_data      = sqlQuery($query, array($appt['pc_aid']));
+$id = $form_id;
+
+?><!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/html">
+<head>
+    <link rel="stylesheet" href="<?php echo $webroot; ?>/interface/main/messages/css/reminder_style.css?v=<?php echo $v_js_includes; ?>" type="text/css">
+    <link rel="stylesheet"  href="<?php echo $GLOBALS['web_root']; ?>/library/css/bootstrap_navbar.css?v=<?php echo $v_js_includes; ?>" type="text/css">
+
+    <?php Header::setupHeader(''); ?>
+    <?php $GLOBALS['medex_enable']='';
```

Comment:
doesn't the above setting of $GLOBALS['medex_enable'] to blank break the code below (ie. it can never be 1)?

Reply:
Testing again for the masses to make it work for everyone via Jitsi8x8 not just MedEx practices.  MedEx people get a different experience.  So I will delete this line.